src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,39 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const booksSlice = createSlice({
+  name: 'books',
+  initialState: {
+    items: [
+      {
+        id: 'item1',
+        title: 'The Great Gatsby',
+        author: 'John Smith',
+        category: 'Fiction',
+      },
+      {
+        id: 'item2',
+        title: 'Anna Karenina',
+        author: 'Leo Tolstoy',
+        category: 'Fiction',
+      },
+      {
+        id: 'item3',
+        title: 'The Selfish Gene',
+        author: 'Richard Dawkins',
+        category: 'Nonfiction',
+      },
+    ],
+  },
+  /* eslint no-param-reassign: ["error", { "props": false }] */
```

Comment:
Hey 👋🏽 seems like you're disabling some rules here because of some linter error, this error is one of the hardest to figure out. ❌  The linter error you're seeing is because you're trying to modify the state object directly inside the `removeBook` reducer. To fix this, you can create a new object with the updated items array instead of modifying the existing `state` object. Here's how you can modify the `removeBook` reducer:
```
removeBook: (state, action) => {
  const itemId = action.payload;
  const updatedItems = state.items.filter((item) => item.id !== itemId);
  return { ...state, items: updatedItems };
},
```
In the updated reducer, we create a new array `updatedItems` that filters out the item with the given `itemId`. Then we return a new object with the original state properties spread using the spread operator `...state`, and overwrite the items property with the `updatedItems` array.

This will achieve your goal of avoiding direct modification of the state object and also fix the linter error. 🤓